lib/LaTeXML/Package/pgfplotstable.sty.ltxml

```diff
@@ -0,0 +1,25 @@
+# -*- CPERL -*-
+# /=====================================================================\ #
+# |  pgfplotstable.sty                                                  | #
+# | Implementation for LaTeXML                                          | #
+# |=====================================================================| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+package LaTeXML::Package::Pool;
+use strict;
+use warnings;
+use LaTeXML::Package;
+
+#**********************************************************************
+InputDefinitions('pgfplotstable', type => 'sty', noltxml => 1);
+# Avoid generic warnings:
+RawTeX('\pgfplotsset{compat=1.17}');
```

Comment:
is this safe, even if the author sets it himself (her)?

Reply:
Assuming the author sets this *after* their `\usepackage{pgfplotstable}`, it should be a clean override. But I can certainly add a check that only sets it if undefined, one minute...

Reply:
a bit of a rabbit hole to find the right macro and figure out its default value (... literally `default`), but I can now claim to have tested this and it only sets the compat mode if it wasn't set before.